cfpb/consumerfinance.gov
# TDP: Add additional document upload fields to support Spanish files

- This PR Updates the ActivityPage model to allow for additional Teacher Guide and Student Handout files
- It also displays the "Activity Type: Available in Spanish" label on search results if that Activity has been tagged with the activity type "Available in Spanish". 


---


## Additions

- Classes ActivityPageActivityDocuments and ActivityPageHandoutDocuments
- Added "Activity Type" field to Search results only when an Activity is tagged "Available in Spanish"



## How to test this PR

1. Edit an Activity Page
2. You'll notice two new fields "Add Additional Activity Files" and "Add Additional Handout Files"
3. Both fields allow for the addition of multiple Documents to the Activity Page. 




## Screenshots
<img width="887" alt="Screen Shot 2021-03-16 at 1 09 58 PM" src="https://user-images.githubusercontent.com/2914091/111352061-2de39400-865a-11eb-8536-137ed1a27053.png">

<img width="791" alt="Screen Shot 2021-03-16 at 1 15 31 PM" src="https://user-images.githubusercontent.com/2914091/111352084-363bcf00-865a-11eb-8596-4e11086ebad0.png">

<img width="888" alt="Screen Shot 2021-03-17 at 3 22 02 PM" src="https://user-images.githubusercontent.com/2914091/113655957-dccb2c80-9668-11eb-9a2d-76c28314d5b3.png">

## Notes and todos

- TODO: We could potentially migrate content from the following existing document fields into these  new fields:
  - Teacher guide
  - Student file 1
  - Student file 2
  - Student file 3


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
